app/api/resources/user.py

```diff
@@ -276,11 +279,22 @@ def post(cls):
         if is_valid != {}:
             return is_valid, 400
 
-        result = DAO.create_user(data)
-
-        if result[1] is 200:
+        failed_send = False
+        try:
             send_email_verification_message(data["name"], data["email"])
 
+        except SMTPException as se:
+            failed_send = True
+            print("SMTP error: {0}".format(se))
```

Comment:
Can we use f-strings here and in the `print()` below, this is so that we have consistency throughout the project.
I listed this here in the [Coding Standards](https://github.com/anitab-org/mentorship-backend/wiki/Coding-Standards) wiki page

Reply:
I removed the print() altogether as we can see it in the console log even without this print()